skl2onnx/operator_converters/support_vector_machines.py

```diff
@@ -140,6 +148,127 @@ def convert_sklearn_svm(scope, operator, container):
         raise ValueError("Unknown support vector machine model type found "
                          "'{0}'.".format(operator.type))
 
+    if (hasattr(op, 'decision_function_shape') and
+            op.decision_function_shape == 'ovr' and handles_ovr):
+        # Applies _ovr_decision_function.
```

Comment:
Wouldn't it be better to just add reference to the scikit implementation rather than adding the entire code here?

Reply:
links to code are not really stable and it is easier to understand what the converter does by looking at the numpy code